report/renderer/report-renderer.js

```diff
@@ -224,6 +224,10 @@ export class ReportRenderer {
           e.preventDefault();
           destEl.scrollIntoView();
         });
+        if (this._opts.anchorTransform) {
+          const newGauge = this._opts.anchorTransform(gaugeWrapperEl);
+          categoryGauge.replaceChild(newGauge, gaugeWrapperEl);
```

Comment:
```suggestion
          gaugeWrapperEl.replaceWith(this._opts.anchorTransform(gaugeWrapperEl));
```

pretty sure this'll do the same deal.. 

however my comment on the dts nullifies this suggestion :)